test/JDBC/upgrade/latest/schedule

```diff
@@ -509,4 +509,5 @@ BABEL-730
 babel-4475
```

Comment:
we should add it to normal JDBC run outside upgrade as well, just adding here is sufficient or we need to add to JDBC schedule as well?

Reply:
This is running in normal JDBC test run. No need to explicitly add it in jdbc_schedule